table.go

```diff
@@ -601,6 +616,32 @@ func (t *Table[V]) All6(yield func(pfx netip.Prefix, val V) bool) {
 	t.rootV6.allRec(nil, false, yield)
 }
 
+// Size returns the sum of the IPv4 and IPv6 refixes.
```

Comment:
"refixes"  :-)  Also applies to the next two comments.

Reply:
Thanks, will be fixed, soon